backend/Contexture.Api/Infrastructure/NStoreBased.fs

```diff
@@ -390,7 +390,31 @@ type Storage(persistence: IPersistence, clock: Clock, logger: INStoreLoggerFacto
             let pollingClient =
                 PollingClient(persistence, positionValue, wrappedSubscription, logger)
 
-            pollingClient.Start()
+            let source = new CancellationTokenSource();
+            let cancellationToken = source.Token;
+            let poll = fun () -> (
+                task {
+                    while not cancellationToken.IsCancellationRequested do
+                        do! pollingClient.Poll().ConfigureAwait(false)
+                        if not cancellationToken.IsCancellationRequested then 
+                            do! Task.Delay(pollingClient.PollingIntervalMilliseconds, cancellationToken).ConfigureAwait(false)
+                    }
+                :> Task
+            )
+
+            Task
+                .Run(poll, cancellationToken)
+                .ContinueWith(fun t ->
+                    if t.IsFaulted then
+                        let innerException = t.Exception.Flatten().InnerException;
+                        let logger = logger.CreateLogger("PollingClient")
+                        logger.LogError($"Error during Poll, first exception: {innerException.Message}.\n{innerException}");
+                        wrappedSubscription.OnErrorAsync(pollingClient.Position ,t.Exception)
+                    else
+                        Task.CompletedTask
+                )
+                .ConfigureAwait(false)
+                |> ignore
```

Comment:
Reimplements polling loop of the `PollingClient` to allow handling errors by the subscription's OnError handler